src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://www.google.com.ua"
+      method="post"
+      class="bigSearchForm"
+      data-qa="keypress">
```

Comment:
check out 5th point from the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)